docs/zh-cn/user/configuration/client.md

```diff
@@ -0,0 +1,202 @@
+---
+title: client
+keywords: 客户端，client provider
+description: 提示用户配置客户端相关配置
+---
+
+# client
+
+## 第一步：编写客户端 `Provider`
+
+1. 编写需要被编码的结构体，由于使用 `Hessian2` 作为编码协议，`User` 需要实现 `JavaClassName` 方法，它的返回值在dubbo中对应User类的类名。
+
+   ```go
+   type User struct {
+   	Id   string
+   	Name string
+   	Age  int32
+   	Time time.Time
+   }
+   
+   func (u User) JavaClassName() string {
+   	return "com.ikurento.user.User"
```

Comment:
please confirm whether the JavaClassName here is change  to "org.apache.dubbo.User",  same with sample “HelloWorld” of verison 1.5